src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,100 @@
+.main {
+  @extend %flex-center;
+}
+
+.page__main {
+  margin-top: 50px;
+}
+
+.catalog-wrapper {
+  @extend %flex-center;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  max-width: $wrapper-width;
+
+  &__card {
+    margin: 0 $margin-between-cards 50px;
+  }
+
+}
+
+.product-card {
+  width: $card-width;
+  padding: 14px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 15px 5px 42px;
```

Comment:
center image using flex on img-wrapper instead of margin with magic numbers (but you'll need margin top and bottom)